docs/guides/1216-docker-cli-load.md

```diff
@@ -3,7 +3,7 @@ slug: /1216/docker-cli-load
 displayed_sidebar: '0.2'
 ---
 
-# Loading a dagger image into a docker daemon
+# Loading an image into a docker engine
 
 Using `cli.#Load`, you can save a dagger image (`docker.#Image`) into a local or remote engine.
```

Comment:
nit: Not sure if the intent was to change all occurrences of `dagger image` to just `image` or just the title, but there is another occurrence here.

Reply:
Oops, not intended, I just didn't think to check the text :) Will fix, thanks.

Reply:
Fixed, I ended up making more edits to the text while I was at it.